src/components/v2/MyAccount/index.tsx

```diff
@@ -0,0 +1,125 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import { Slider } from '../Slider';
+import { useMyAccountStyles as useStyles } from './styles';
+import { IToggleProps, Toggle } from '../Toggle';
+import { Icon } from '../Icon';
+import { Tooltip } from '../Tooltip';
+
+function toPercentageString(n: number) {
+  return `${n}%`;
+}
+function toUsdString(n: number) {
+  return `$${n}`;
```

Comment:
This util should also divide the value passed by 100, to convert it into dollars (since you're passing values in cents to it).